app/tests/test_queries.py

```diff
@@ -584,6 +584,462 @@ def test_browse_with_date_from_and_to_filter_no_result(
 
         assert result.items == []
 
+    def test_browse_with_transferring_body_sorting_a_to_z(
```

Comment:
Like we discussed before, these tests could possibly be simpler and as effective at a lower level by moving the execution of the query (using pagination) to the route itself. Then we could test all these sorting edge cases without needing to worry about pagination and requiring lots of test objects.

This can be addressed when we tackle the ticket to break out the body route into distinct routes

Reply:
yes, I agree , we should address when we create distinct routes